src/tribler/core/components/libtorrent/download_manager/download.py

```diff
@@ -123,8 +128,8 @@ def get_torrent_data(self) -> Optional[object]:
     def register_alert_handler(self, alert_type: str, handler: lt.torrent_handle):
         self.alert_handlers[alert_type].append(handler)
 
-    def wait_for_alert(self, success_type: str, success_getter: Optional[Callable[[Any], Any]] = None,
-                       fail_type: str = None, fail_getter: Optional[Callable[[Any], Any]] = None) -> Future:
+    def wait_for_alert(self, success_type: str, success_getter: Getter = None,
```

Comment:
**nitpick:** It may be easier to understand the function signature if, in the type declaration, `Getter` was declared as `Callable[[Any], Any]` (without `Optional`). Then, we could have `success_getter: Optional[Getter] = None` in the function signature.

Reply:
Changed